src/fake_data/provide_hass.ts

```diff
@@ -224,6 +224,7 @@ export const provideHass = (
       language: localLanguage,
       number_format: NumberFormat.language,
       time_format: TimeFormat.language,
+      first_weekday: Weekday.monday,
```

Comment:
```suggestion
      first_weekday: Weekday.language,
```
if default, use:
`"weekInfo" in Intl.Locale.prototype ? getLocaleFirstWeekDay(language) : Weekday.monday`

Reply:
I don't think we can do this. If user select "language", it will have the right day on compatible browser and monday on other (firefox).

Reply:
We can fallback to this package if needed https://github.com/gamtiq/weekstart. But I think the best solution is to define Monday for everyone and let the user choose in profile.

Reply:
weekstart is pretty small, I think we can use that as fallback?

Reply:
Yep, I just tested it and it don't impact the app size.